package.json

```diff
@@ -58,6 +57,7 @@
         "sinon": "^14.0.0",
         "supertest": "4.0.2",
         "supertokens-node": "file:../supertokens-node",
```

Comment:
yes, at least until we migrate all tests

Reply:
what about after the migration?

Reply:
for development it must be "file:../supertokens-node" so we can test the branch we are working on.

for CICD it doesn't work because the project is cloned in a folder called "project", I had to add an extra line there to make it work. https://github.com/supertokens/supertokens-node/pull/849/files#diff-914fc53a25e9e22dda7ca389054b946199433a75bd11548983e0a62615ed15a7R75 

any ideas on how to handle it?

Reply:
why do we need it during development, for types only?
as far as I understand for running the test in CI we do not necessarily need (or want) to have the current version linked: 
for example if `convertToRecipeUserId` changes for some reason and no longer supports the signature we use to call it in the tests, we may have to backport those updates to branches testing older FDIs and support running with both the new version and old version of the node sdk... which can be bad.
All in all: it should have a "normal" version set in the dependencies and maybe a script or something to link for local development.

Reply:
ok